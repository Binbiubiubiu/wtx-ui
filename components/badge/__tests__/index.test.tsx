import React from "react";
import { render } from "enzyme";
import toJson from "enzyme-to-json";
import Badge from "..";

describe("<Badge />", () => {
  it("should render a <Badge/> components", () => {
    const wrapper = render(
      <div>
        <Badge>默认</Badge>
      </div>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
