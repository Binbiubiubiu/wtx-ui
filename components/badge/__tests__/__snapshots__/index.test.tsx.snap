// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> should render a <Badge/> components 1`] = `
<div>
  <div
    class="wtx-badge wtx-badge__default"
  >
    默认
  </div>
</div>
`;
